### PR TITLE
Update js samples

### PR DESCRIPTION
**Pull request by aiham**
- Use latest JS API
- Use best practices: error handling
- Make JS style uniform
- Use HTTPS for JS URL

### CODE REVIEW

**Comment by aiham** [comment]
Resolves #60

File: Samples/Archiving/Content/js/participant.js [hunk 1]
```diff
@@ -1,13 +1,24 @@
-var session = OT.initSession(sessionId),
-    publisher = OT.initPublisher("publisher");
+var session = OT.initSession(apiKey, sessionId),
+    publisher = OT.initPublisher('publisher');
 
-session.connect(apiKey, token, function(err, info) {
-  if(err) {
-    alert(err.message || err);
+session.connect(token, function(error, info) {
```
Comment (aullman) [inline Samples/Archiving/Content/js/participant.js]: `info` isn't used and it's not actually in our API. Let's remove it.

File: Samples/Archiving/Content/js/host.js [hunk 2]
```diff
@@ -1,50 +1,61 @@
-var session = OT.initSession(sessionId),
-    publisher = OT.initPublisher("publisher"),
+var session = OT.initSession(apiKey, sessionId),
+    publisher = OT.initPublisher('publisher'),
     archiveID = null;
 
-session.connect(apiKey, token, function(err, info) {
-  if(err) {
-    alert(err.message || err);
+session.connect(token, function(error, info) {
```
Comment (aullman) [inline Samples/Archiving/Content/js/host.js]: Let's remove `info` here as well.

**Comment by aullman** [comment]
I submitted https://github.com/opentok/Opentok-.NET-SDK/pull/80 to remove the info parameters.

**Pull request merged**